go-controller/pkg/ovn/ovn.go

```diff
@@ -716,43 +719,59 @@ func (oc *Controller) WatchCRD() {
 func (oc *Controller) WatchEgressFirewall() *factory.Handler {
 	return oc.watchFactory.AddEgressFirewallHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
-			egressFirewall := obj.(*egressfirewall.EgressFirewall).DeepCopy()
-			errList := oc.addEgressFirewall(egressFirewall)
-			for _, err := range errList {
-				klog.Error(err)
-			}
-			if len(errList) == 0 {
-				egressFirewall.Status.Status = egressFirewallAppliedCorrectly
+			egressFirewall := obj.(*egressfirewallapi.EgressFirewall).DeepCopy()
+			// TODO: when addressSet becomes its own package make sure to check
```

Comment:
Good Catch, no and the wording for what I was thinking is different then what I want to actually happen. In this case the egressFirewall is just using the hashed name of the addressSet in a logical router policy match statement. since we are not adding any IPs to the set it does not matter when the set is actually created as long as the name is known which it is becase they get generated the same way.

Reply:
done